api/product.rb

```diff
@@ -91,6 +91,24 @@ def set_properties_based_on_version(version)
     end
     # rubocop:enable Naming/AccessorMethodName
 
+    def exists_at_version_or_lower(name)
+      return true if @versions.nil?
+      name ||= 'ga'
```

Comment:
might be worth using `Version::ORDER[0]` as the fallback

Reply:
Done